spec/airbrake_spec.rb

```diff
@@ -167,11 +167,11 @@
       filter_chain = notifier.instance_variable_get(:@filter_chain)
       filters = filter_chain.instance_variable_get(:@filters)
 
-      expect(filters.size).to eq(2)
+      expect(filters.size).to eq(3)
```

Comment:
That's expected https://github.com/airbrake/airbrake-ruby/blob/43f780b2053532349056fb06e9814ea203beba4a/lib/airbrake-ruby/filter_chain.rb#L56-L57